### DISCUSSION TITLE
Trivy DB Source URL (for Air-gapped / URL Whiltelisting Environments)

### DISCUSSION THREAD

**Discussion by 7sn** [aquasecurity/trivy#1499] [Q&A]
Hi,

I'm Setting up Harbor with Trivy Security Scanner. My setup is air-gapped (internet is allowed only on certain URLs per node) and as such we need to know the source URL of the Security Scanner.

**Comment by AndreyLevchenko** [comment]
Hi Hasan
Thank you for your question.
Please check: https://aquasecurity.github.io/trivy/v0.21.3/advanced/air-gap/
